pyproject.toml

```diff
@@ -83,7 +83,14 @@ skip = [
 # Ignore table where words could be split across rows
 ignore-regex = '(\|.*\|.*\|.*\||\[[A-Z][a-z]\][a-z][a-z])'
 ignore-words-list = [
-  "CrossReference"
+  "CrossReference",
+  "ND",
+  "MILIARY",
+  "Miliary",
```

Comment:
could you add a line into one of your upcoming PRs to mention that these two words are from the phenopackets schema?